pkg/api/types.go

```diff
@@ -1362,6 +1362,8 @@ type PodSecurityContext struct {
 	// Use the host's ipc namespace.
 	// Optional: Default to false.
 	HostIPC bool `json:"hostIPC,omitempty"`
+	// The SeccompProfile is the seccomp profile this pod should run under.
```

Comment:
Will update!

On Monday, May 9, 2016, Paul Morie <notifications@github.com> wrote:

> In pkg/api/types.go
> <https://github.com/kubernetes/kubernetes/pull/25324#discussion_r62559109>
> :
>
> > @@ -1362,6 +1362,8 @@ type PodSecurityContext struct {
> >  	// Use the host's ipc namespace.
> >  	// Optional: Default to false.
> >  	HostIPC bool `json:"hostIPC,omitempty"`
> > +	// The SeccompProfile is the seccomp profile this pod should run under.
>
> This should be a name right now to match the proposal, I do not want to
> specify these profiles inline -- it makes the policy much harder to write.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25324/files/ec54b7e13bbd79087fb34f0973f4482c0150c9c6#r62559109>
>


-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>